### PR TITLE
fix(ui): missed anchor target effect for activity

### LINKED ISSUE #202: Fix effect for activity on email redirected by link
**Problem**
On Emails tab, in Email card there is time link. By clicking on this link you'll be redirected to the corresponding event in Activities tab, but the activity is not highlighted (previously was highlighted via light blue color).

CleanShot 15 11 2024 at 23-24

CleanShot 15 11 2024 at 23-26

**Solution**
Apply tblr-blue-lt (#E6F1FA) for background-color for row with event when redirected by link in the Email card.

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #202.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/controllers/ats/candidates_controller.rb [hunk 1]
```diff
@@ -134,6 +134,13 @@ def show
         when "files"
           @all_files = @candidate.all_files
         when "activities"
+          if params[:event]
+            redirect_to tab_ats_candidate_path(@candidate, :activities,
+                                               page: page_of_activity(params[:event]),
+                                               anchor: "event-#{params[:event]}")
+            return
+          end
```
Comment (ksnitsky) [inline app/controllers/ats/candidates_controller.rb:142]: Reduced number of queries

**Comment by wrmk** [comment]
got this error on trying to open activity via link from emails tab
image

**Review by wrmk — COMMENTED** [review]
see comment

**Comment by wrmk** [comment]
> Didn't come up with a quick solution, reverted the code back. Ideally it should be fixed, because when you click on a link almost all logic is executed 2 times because of redirection

please create low prior issue

**Comment by ksnitsky** [comment]
https://github.com/freeats/freeats/issues/212

**Pull request merged**